x-pack/plugins/security_solution_serverless/server/task_manager/usage_reporting_task.ts

```diff
@@ -159,11 +169,40 @@ export class SecurityUsageReportingTask {
 
     const state = {
       lastSuccessfulReport:
-        usageReportResponse?.status === 201 ? new Date() : taskInstance.state.lastSuccessfulReport,
+        usageReportResponse?.status === 201
+          ? meteringCallbackTime
+          : this.getFailedLastSuccessfulReportTime(
+              meteringCallbackTime,
+              taskInstance.state.lastSuccessfulReport,
+              lookBackLimitMinutes
+            ),
     };
     return { state };
   };
 
+  private getFailedLastSuccessfulReportTime(
+    meteringCallbackTime: Date,
+    lastSuccessfulReport: Date,
+    lookBackLimitMinutes?: number
+  ): Date {
+    const nextLastSuccessfulReport = lastSuccessfulReport || meteringCallbackTime;
+
+    if (!lookBackLimitMinutes) {
+      return nextLastSuccessfulReport;
+    }
+
+    const lookBackLimitTime = new Date(meteringCallbackTime.setMinutes(-lookBackLimitMinutes));
+
+    if (nextLastSuccessfulReport > lookBackLimitTime) {
+      return nextLastSuccessfulReport;
+    }
+
+    this.logger.error(
+      `lastSuccessfulReport time of ${nextLastSuccessfulReport.toISOString()} is past the limit of ${lookBackLimitMinutes} minutes, adjusting lastSuccessfulReport to ${lookBackLimitTime.toISOString()}`
```

Comment:
I am seeing PR failures that it seem it is possible for `nextLastSuccessfulReport` to be undefined: https://buildkite.com/elastic/kibana-pull-request/builds/161779#018acc9c-6d0c-42c3-b1da-62764b1a923b/300-4161

Can we fix that?